samples/Samples.HttpMessageHandler/WebClientHelpers.cs

```diff
@@ -0,0 +1,334 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Datadog.Trace;
+
+namespace Samples.HttpMessageHandler
+{
+    internal static class WebClientHelpers
```

Comment:
Duplicate of `samples/Samples.HttpClientDriver/WebClientHelpers.cs`?

Reply:
removing.